### PR TITLE
Org teams refactor

### PR DESCRIPTION
**Pull request by future-pirate-king**

### CODE REVIEW

File: app/components/organization-team/add-team-member/index.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.emtpy-container {
```
Comment (Yibaebi) [inline app/components/organization-team/add-team-member/index.scss:1]: Some of the declarations here can be replaced with our utility classes. Don't you think?

Comment (future-pirate-king) [reply app/components/organization-team/add-team-member/index.scss:1]: yeah but I would still require the class so keeping everything together

File: app/components/organization-team/add-team-project/index.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.emtpy-container {
```
Comment (Yibaebi) [inline app/components/organization-team/add-team-project/index.scss:1]: The utility classes can still be used in this scenario IMO.

File: app/components/organization-team/details/team-info/index.hbs [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+          <AkIcon @iconName='delete' />
+        </:leftIcon>
+
+        <:default>{{t 'deleteTeam'}}</:default>
+      </AkButton>
+    </AkStack>
+
+    <ConfirmBox
+      @isActive={{this.showTeamDeleteConfirm}}
+      {{!-- @title={{t 'promptBox.deleteTeamPrompt.title'}} --}}
```
Comment (Yibaebi) [inline app/components/organization-team/details/team-info/index.hbs:108]: Comment?

File: app/components/organization-team/invitation-list/index.hbs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      data-test-teamInviteList-description
+      @color='textSecondary'
+      @variant='body2'
+    >
+      {{t 'teamInvitationsDesc'}}
+    </AkTypography>
+  </AkStack>
+
+  <AkStack @spacing='1'>
+    {{!-- <div {{style width='180px'}}>
```
Comment (Yibaebi) [inline app/components/organization-team/invitation-list/index.hbs:17]: Commented code?

File: app/components/organization-team/member-list/index.hbs [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+      data-test-teamUserList-description
+      @color='textSecondary'
+      @variant='body2'
+    >
+      {{t 'teamUsersDesc'}}
+    </AkTypography>
+  </AkStack>
+
+  <AkStack @spacing='1'>
+    {{!-- <div {{style width='180px'}}>
```
Comment (Yibaebi) [inline app/components/organization-team/member-list/index.hbs:18]: Commented code.

File: app/components/organization-team/invitation-list/index.hbs (hunk 4 above)
Comment (future-pirate-king) [reply app/components/organization-team/invitation-list/index.hbs:17]: at the api does not support search so will uncomment once backend change is there

File: app/components/organization-team/member-list/index.hbs (hunk 5 above)
Comment (future-pirate-king) [reply app/components/organization-team/member-list/index.hbs:18]: same reason as above

File: app/components/organization-team/project-list/access-permission/index.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+export default class OrganizationTeamProjectListAccessPermission extends Component {
+  @service intl;
+  @service('notifications') notify;
+
+  tPleaseTryAgain = this.intl.t('pleaseTryAgain');
+  tPermissionChanged = this.intl.t('permissionChanged');
+
+  @task *changeProjectWrite() {
+    try {
+      const prj = this.args.project;
```
Comment (Yibaebi) [inline app/components/organization-team/project-list/access-permission/index.js:14]: I would favour "project" over "prj".

Comment (Yibaebi) [reply app/components/organization-team/project-list/access-permission/index.js:14]: This is irrelevant though.

**Pull request merged**